tests/integration/it_authorization_test.py

```diff
@@ -4,18 +4,15 @@
 from tests import factories
 from timeless.access_control.authorization import is_allowed
 from timeless.access_control.methods import Method
+from timeless.roles.models import RoleType
 
 
-# @todo #329:30min After #298 is finished update this test cases with
-#  appropriate roles and make sure that current test cases diffrentiate enough
-#  between roles, so we can be sure that is_allowed takes into account all
-#  exiting roles. Also remove pytest.mark.skip from all test methods here.
-#  And also try to parametrize tests below if it will be possible, at least 3
-#  of those tests are very similar (tests with just a single company) so it
-#  should be easy to parametrize them.
 def test_manager_can_access_employee():
     my_company = factories.CompanyFactory()
-    me = factories.EmployeeFactory(company=my_company)  # set role to manager
+    manager_role = factories.RoleFactory(
+        name="manager", role_type=RoleType.Manager)
```

Comment:
@vladarefiev I'm not confortable with these hardcoded strings for roles. I think that we should make constants to these values in `Role` or an enumeration for these strings, WDYT?